pontoon/base/models.py

```diff
@@ -116,6 +138,14 @@ def stringify(self):
     class Meta:
         ordering = ['name', 'code']
 
+    def latest_activity(self, project=None):
```

Comment:
If you move the template logic mentioned below into the view, these model methods become less useful. I think it'd be better to go without them and move this logic into the view as well.